src/sw/redis++/utils.h

```diff
@@ -92,7 +97,7 @@ using Optional = std::optional<T>;
 template <typename T>
 class Optional {
 public:
-    Optional() = default;
+    Optional(): _value() {};
```

Comment:
This is used for fixes crash on MSVC2015

Reply:
Is this a MSVC bug? Really strange. This should be valid C++ code.

Reply:
Yeap, MSVC bug